test/assume-role.bats

```diff
@@ -47,6 +47,12 @@ setup() {
 }
 
 teardown() {
+  # This will output the if the test fails
+  for i in ""${!lines[@]}""
```

Comment:
What are these double'd double quotes doing?

Reply:
should be single quotes

Reply:
fixed